### DISCUSSION TITLE
Grafana stopped displaying charts

### DISCUSSION THREAD

**Discussion by hufhend** [prometheus/snmp_exporter#959] [Q&A]
### Host operating system: output of `uname -a`
docker: Linux d369f3590257 5.16.17-sun50iw9 #3.0.6 SMP Tue Aug 9 13:51:16 CST 2022 aarch64 GNU/Linux

### snmp_exporter version: output of `snmp_exporter -version`

snmp_exporter, version 0.23.0 (branch: HEAD, revision: 20657d2f486a1ac237238827beed97fa67eb8611)
  build user:       root@e4de29736f81
  build date:       20230816-09:52:12
  go version:       go1.20.7
  platform:         linux/arm64
  tags:             netgo

### What device/snmpwalk OID are you using?
Mikrotik

### What did you do that produced an error?
I promoted to the new version, or rather the promotion was done by crontab

### What did you expect to see?
I expected it to go further

### What did you see instead?
Everything broke. First the message in the log pointed me to a new configuration file, I generated it. Then I searched for settings for auth=public_v3, but so be it.
I repeated snmpwalk, tried grabbing the data via snmp-exporter as well, edited the configuration in Prometheus, checked that Prometheus sees the tagret fine, and that leaves Grafana. 
I wouldn't expect any more changes here, I use [this dashboard](https://grafana.com/grafana/dashboards/14857-mikrotik-snmp/)  and I suspect something must have changed in the instance variable as well. 
I don't see any charts at all, nor do I have the option of selecting the correct instance. Does anyone know what to do about this?

**Comment by SuperQ** [comment]
Please read the release notes before upgrading.

**Comment by hufhend** [comment]
I read it already during the search and now again, but unfortunately there is nothing to Grafana and even the documentation does not help. 
Apparently something has changed and I may be one of the first to come across it. Wouldn't it be worth describing a way to achieve the original functionality at least for Grafana?
